### ISSUE TITLE
Meet with collaborators to ensure the schematic is accurate

### ISSUE THREAD

**Issue by RobertGregg** [RobertGregg/SepsisProject#2]
┆Board Name: Sepsis Modeling

**Comment by RobertGregg** [comment]
➤ Rob Gregg commented:

* Relationship between HIF1 and AMPK
    * We have found conflicting evidence for HIF1 and AMPK both activating and inhibiting each other
      * think it depends on the context, for sepsis what we have in the diagram is good
      * PKM2 is an enzyme that effects pyruvate (another important key to Warburg effect)
      * HIF1a is stabilized by inflammation

  * Addition of ROS into the model
    * Good addition, adds some interesting feedback loops into the model.
    * mitochondria ROS vs cytosolic ROS? 

  * Three ways that AMPK can get activated
    * Calcium calmodulin kk (activates AMPK)
    * lkb1 (canonical activation of AMPK)
    * ATP/AMP ratio
      * Could incorporate these into the model to see how they influence HIF1a dynamics

  * How dies HIF1 get activated via the LPS/PI3K pathway?
    * Carlos will send a paper about septic shock
    * Basically LPS enters the bloodstream, gets filtered through the kidney, and enters tubular epithelial cells via TLR4
